### ISSUE TITLE
Improve text bubbles functionality to facilitate sticky note exercises

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2485]
**Is your feature request related to a problem? Please describe.**
Community members frequently mention they would like to be able to do sticky note exercises within hubs. The current text bubbles come close, but don't quite provide the types of features that would be helpful for this type of work. 

** Some suggestions identified by Kip and Roland at Mozilla  **
Sticky notes which I can write/type on for group ideation work.
Writeable cards, for card sorting exercisess 
Voting mechanic for sticky notes and cards.
Dictaphone, less intensive than video for recording.
Simple 2D drawing tool on a specific "paper" surface.
Private objects, for when I want to import my presentation notes without the world seeing them.
A means to send all notes, drawings and recordings to the admin's email address, or for them to be downloaded.

** The community has also mentioned being able to **
Change color of text bubbles for improved sorting
Enabling users to edit text bubbles
Connect to a tool like Miro

**Comment by Anandavadivelan** [comment]
Yes I think sticky notes with speech to text will be very useful for collaborative work in hubs meetings

**Comment by Lucero-brilliante** [comment]
It would be great to have the features of e.g. mural boards in hubs. Facilitation would be possible on a much higher level.
